test/index.html

```diff
@@ -125,6 +128,18 @@
         });
       }
 
+      const Plugin = <div />;
+      const viewer = function(results) {
```

Comment:
This is the function that takes in the results as a string. Just like the `value` in `ResultsViewer` [here](https://github.com/graphql/graphiql/blob/master/src/components/ResultViewer.js#L25)